src/main/java/frc/robot/Constants.java

```diff
@@ -9,6 +9,7 @@
 import edu.wpi.first.math.geometry.Transform3d;
 
 public final class Constants {
+
```

Comment:
accidental keystroke?

Reply:
The new line after `Constants {`? Intentional, just helps add some breathing room to the file - although that's definitely a personal preference thing.